vendor/src/github.com/docker/libcontainer

```diff
@@ -0,0 +1 @@
+../../../../
```

Comment:
hmm i think we dont need this. If the desired state is to have a compiled nsinit binary, there's `make binary` which installed nsinit binary into `./bundle`